README.rst

```diff
@@ -248,6 +249,16 @@ keys:
     timestamp, the configuration will override the value in the state file, and
     the next ``tinuous`` run will only retrieve assets after the new setting.
 
+``max-days-back``
+    An integer specifying the maximum number of days back to look for builds;
+    defaults to 30.
+
+    If ``since`` is earlier than the date indicated by this value, and if the
+    timestamp for a given CI system saved in the state file is either missing
+    (i.e., if this is the first run of ``tinuous`` against the CI system) or
+    older than ``since``, then ``tinuous`` will use the explicitly-specified
+    ``since`` value as the cutoff and ignore ``max-days-back``.
```

Comment:
had to re-read a number of times to fit it into mental logical operation ... hopefully I got it right ;)